### PR TITLE
Add arXiv:1709.10106 to the paper list

### PR DESCRIPTION
**Pull request by bostdiek**

### CODE REVIEW

**Comment by matthewfeickert** [comment]
@bostdiek Thanks for the PR!

Overall everything looks good, but I see that there were a few changes to the `README` structure overall. I suspect this is because you actually followed the `CONTRIBUTING` instructions and beautified the Markdown. I know that this is what I currently ask people to do, but Tidy Markdown has actually become too aggressive, and I am looking for a different solution.

Unless you have other specific edits to the `README` structure that you think we should make can you reset the `README` to the the `master` `HEAD` and then just readd your contribution of the paper?

File: README.md [hunk 1]
```diff
@@ -180,10 +193,11 @@ the CMS detector in pp collisions at 13 TeV](https://inspirehep.net/record/16443
 
 - T. Cheng, "[Recursive Neural Networks in Quark/Gluon Tagging](https://inspirehep.net/record/1634876)," arXiv:1711.02633 [hep-ph]. (November 7, 2017)
 
-- M. Frate, K. Cranmer, S. Kalia, A. Vandenberg-Rodes, and D. Whiteson, “[Modeling Smooth Backgrounds and Generic Localized Signals with Gaussian Processes](https://inspirehep.net/record/1624168),” arXiv:1709.05681 [physics.data-an]. (September 17, 2017)
+- S. Chang, T. Cohen, and B. Ostdiek, "[What is the Machine Learning?](https://inspirehep.net/record/1627883)," arXiv:1709.10106 [hep-ph]. (September 28, 2017)
```
Comment (matthewfeickert) [inline README.md]: @bostdiek Unless you have other specific edits to the `README` structure that you think we should make can you reset the `README` to the the master `HEAD` and then just readd this contribution?

**Comment by matthewfeickert** [comment]
@bostdiek Actually, good catch on the `####` needing to be `###` --- not sure how that got messed up in the first place. If you can leave that formatting in as well that'd be great. :+1:

File: README.md (hunk 1 above)
Comment (bostdiek) [inline README.md]: @matthewfeickert All of the changes (other than adding the paper) were made by tidy-markdown. I will make a new version not using that will also incorporate the changes in the headings which you mentioned.

Comment (bostdiek) [reply README.md]: @matthewfeickert I think these should be in the correct format now

Comment (matthewfeickert) [reply README.md]: LGTM! :shipit:

**Comment by matthewfeickert** [comment]
Many thanks for the contribution, @bostdiek!

**Pull request merged**